INSTALL.md

```diff
@@ -37,7 +37,7 @@ make bash
 All the source and dependencies for this project will be in place, allowing you to run the server and the tests from inside the container:
 ```bash
 $ bundle exec rake integrate # Runs the test suite
-$ APICAST_REGISTRY_URL=https://apicast-staging.proda.3sca.net/policies UNICORN_WORKERS=2 rails server -b 0.0.0.0 # Runs the server, available at localhost:3000
+$ UNICORN_WORKERS=2 rails server -b 0.0.0.0 # Runs the server, available at localhost:3000
```

Comment:
Unrelated but at this point I'd really recommend:
```suggestion
$ UNICORN_WORKERS=8 rails server -b 0.0.0.0 # Runs the server, available at localhost:3000
```

Reply:
Is this still valid?

Reply:
@josemigallas The change of `2` for `8` yes, it is unrelated to the PR, it is just documentation of Porta in general that I think it is a better recommendation. But this admits some subjectivity I suppose 😄

Reply:
OK change proposed in #758